src/dev/app.md

```diff
@@ -29,15 +31,63 @@ docker pull cozy/cozy-app-dev
 
 (We update this image on a regular basis with the latest version of the server and our library. Don’t forget to update the image by running `docker pull cozy/cozy-app-dev` from time to time).
 
+## Create your application
 
-## Create your first application
+You can boostrap your application from scratch if you want, but we recommand to use our new community tool [`create-cozy-app`](https://github.com/CPatchane/create-cozy-app) to bootstrap very easily a Cozy application for you.
+
+This tool will generate an application using (P)React, the framework we internally use in the Cozy Front team. But [options](https://github.com/CPatchane/create-cozy-app#options) are available if you want to use other frameworks.
+
+!!! warning ""
+    For now the new [`cozy-client`](https://github.com/cozy/cozy-client) is used only in the (P)React template (it doesn't use the previous `cozy-client-js` anymore). This library is at an early stage but you can use it, it will be our next Cozy client for application development.
+
+First of all, run directly `create-cozy-app` without installing it globally by using the `yarn create cozy-app` command to bootstrap your application:
+
+```
+yarn create cozy-app mycozyapp
+```
+
+The script will download some dependencies (may take a while) and ask you a few questions, then create an application skeleton inside `mycozyapp`.
```

Comment:
This is when a tool is asking questions that the developer should be accompanied.
Otherwise you might as well write a bash script with the different commands to run.